### PR TITLE
[IMP]10718 add module web_list_view_sequence.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: web_list_view_sequence/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_list_view_sequence/__init__.py:1]: headers/readme

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991

**Comment by siyuancn** [comment]
LGTM

**Comment by elicoidal** [comment]
LGTM

**Pull request merged**